openstack/nova/values.yaml

```diff
@@ -86,13 +90,21 @@ portMetrics: '9102'
 
 sentry_dsn: DEFINE_IN_REGION_CHART
 
+# Use loci only for neutron
+loci:
+  neutron: true
+  nova: false
+
 #TODO we need to move to global or find another way to share image versions
+imageNameNeutron: loci-neutron
+imageVersionNeutronServerM3: null
 imageVersionNeutronServer: null
 imageVersionNeutronOpenvswitchAgent: null
-imageVersionOpenvswitchVswitchd: null
-imageVersionOpenvswitchDbServer: null
+imageVersionNeutronOpenvswitchVswitchd: null
```

Comment:
The openvswitchd is not an neutron image, it contains simply the vswitchd, hence the name.
The image from kolla is called openstack-openvswitch-vswitchd, because there is no neutron code in there.

Reply:
openvswitchd isn't used from kolla anymore

Reply:
Your template suggests, that it is an option to use either kolla or loci images for the openvswitch parts.

If you want to only support the loci images there, remove the option.
If you want to support the change, you would need to fix the image-version attributes.

Reply:
you're right. there is no kolla support for neutron-related containers. I will remove it

Reply:
I removed "kolla" support for neutron related containers since they would probably won't work anyway